lib/Form/DisplayFormInWPContent.php

```diff
@@ -20,8 +20,11 @@ public function __construct(WPFunctions $wp, FormsRepository $formsRepository) {
     $this->formsRepository = $formsRepository;
   }
 
-  public function display(string $content): string {
-    if(!$this->shouldDisplay()) return $content;
+  /**
+   * @return string|null
+   */
+  public function display(string $content = null) {
```

Comment:
We should be safe and remove the type-hint all together here and check if the value is a string.